navigator/runtime/src/main/java/com/freeletics/mad/navigator/NavRoute.kt

```diff
@@ -56,7 +56,7 @@ public interface ExternalActivityRoute : ActivityRoute {
 /**
  * Returns the [ActivityRoute] that was used to navigate to this [Activity].
  */
-public fun <T : ActivityRoute> Activity.requireRoute(): T {
+public fun <T : InternalActivityRoute> Activity.requireRoute(): T {
```

Comment:
It doesn't make sense on `ActivityRoute` because we don't add the route to the intent for `ExternalActivityRoute`. Also since it is going to another app that app wouldn't even have acces to the route class.